WebUI/lims/views.py

```diff
@@ -255,15 +258,25 @@ def DeleteNotes(request, studyId, id):
 def createDatabase(request):
 
     # Prepare the connection string
-    username = ''    # TODO get the username from the form
-    password = ''    # TODO Get password from the form
+    username = request.POST['userName']    # TODO get the username from the form
```

Comment:
Once the task is done you can remove the TODO

Reply:
I will remove them